common-money.lic

```diff
@@ -29,7 +29,11 @@ module DRCM
   end
 
   def wealth(hometown)
-    currency = get_data('town')[hometown]['currency']
```

Comment:
This feels a little hacky to me. Could we have two methods (and the one that looks up your hometown currency could call the other one)?